content/post/104-dht-deep-dive.md

```diff
@@ -0,0 +1,150 @@
+---
+date: 2020-07-20
+url: /2020-07-20-dht-deep-dive/
+title: IPFS 0.5 Content Routing Improvements: Deep Dive
```

Comment:
@momack2 The error is complaining about this line... I suspect the colon in `Improvements: Deep` is being interpretreted as attepting to write a nested object.

```
Error: Error building site: "/home/circleci/project/content/post/104-dht-deep-dive.md:4:1": failed to unmarshal YAML: yaml: line 3: mapping values are not allowed in this context
 ```

Reply:
I think wrapping the the title in quotes `"` should fix it

```suggestion
title: "IPFS 0.5 Content Routing Improvements: Deep Dive"
```